tools/db_bench_tool.cc

```diff
@@ -3711,6 +3751,18 @@ void VerifyDBFromDB(std::string& truth_db_name) {
       }
       thread->stats.FinishedOps(db_with_cfh, db_with_cfh->db,
                                 entries_per_batch_, kWrite);
+      if (FLAGS_sine_write_rate) {
+        uint64_t now = FLAGS_env->NowMicros();
+        int64_t usecs_since_last = static_cast<int64_t>(now - thread->stats.GetSineInterval());
```

Comment:
This cast is a little bit confusing. The only place `usecs_since_last` can be negative is the case that `now - thread->stats.GetSineInterval()` overflows. So we are under the assumption that we can get a correct negative value after casting from an overflowed value. Though in reality it may help in some special case where clock is set back, it still feels confusing to me.
My suggestion is to make `usecs_since_last` uint64_t, and set it to 0 if `now` is smaller than thread->stats.GetSineInterval().

Reply:
Okay, I was kind of thinking that the value was only increasing, but should be handled anyways in case of the variables being changed as you say. The point was to avoid unsigned and signed comparison, since the interval flag was signed.